trigger_circleci_pipeline.sh

```diff
@@ -99,3 +99,27 @@ for workflow_id in $pipeline_workflows_response
 do
   echo "${CIRCLE_WORKFLOW_URL_BASE}/${workflow_id}"
 done
+
+if [ "${SHOULD_WAIT}" = "true" ]; then
+    echo "Waiting for workflows to finish"
+    for workflow_id in $pipeline_workflows_response
+    do
+        while :; do
+            sleep 10
+            result=$(curl -sSX GET -H "Circle-Token: ${CIRCLECI_USER_PERSONAL_API_TOKEN}" ${CIRCLECI_API_BASE}/workflow/${workflow_id} | jq ".status")
```

Comment:
Hope the API is safe, because one timeout and you might think the job as failed 🤔

Reply:
only time will tell, if it isn't stable we'll change the code accordingly, but it's hard to tell without any failures :)